### PR TITLE
Added NGINX Documentation (Fix)

### PR DESCRIPTION
**Pull request by RyoshiKayo**
Added Fixed NGINX Documenation with proper tabs, and linted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,51 +23,211 @@ const db = require('knex')(config.database)
...
+
+```nginx
+server {
+    listen  subdomain.domain.com:80;    # Listen to IPv4 traffic on port 80.
+
+    server_name subdomain.domain.com;   # The domain you set in your lolisafe config.js file.
+
+    root    /path/to/local/lolisafe/github/repo/pages;  # NOTE! The path MUST end with '/pages'.
+
+    index   home.html;              # This is the file to be sent to the requested user when they go to your domain (You shouldn't need to change this).
```
Comment (alucard0134) [inline README.md]: not necessary imo, just / always is fine

File: README.md [hunk 2]
```diff
@@ -20,51 +23,211 @@ const db = require('knex')(config.database)
...
+
+    client_max_body_size    512M;   # This must be grater than, or equal to the max upload size you set in the lolisafe config.
+
+    access_log  /var/log/nginx/subdomain.domain.com.access.log;     # The file where all NGINX requests are logged.
+    error_log   /var/log/nginx/subdomain.domain.com.error.log;      # The file where all NGINX errors are logged.
+
+    location / {
+        proxy_pass  http://subdomain.domain.com:9999;   # This is the domain and port number you set in the lolisafe config.js file.
```
Comment (alucard0134) [inline README.md]: 127.0.0.1?

File: README.md [hunk 3]
```diff
@@ -20,51 +23,211 @@ const db = require('knex')(config.database)
...
+        proxy_set_header    x-Forwarded-For     $proxy_add_x_forwarded_for;
+        proxy_set_header    x-Forwarded-Proto   $scheme;
+    }
+}
+
+# Close vi by pressing ESC + :q! + ENTER.
+```
+
+1. Stage your config file to be live. This creates a [symbolic link](https://kb.iu.edu/d/abbe) to the configuration we just edited, so you only have to edit the file in `/etc/nginx/sites-available`
+    - `sudo ln -s /etc/nginx/sites-available/subdomain.domain.com /etc/nginx/sites-enabled/`
```
Comment (alucard0134) [inline README.md]: n-no

fucking no

nginx just looks for vhost files since 1.10

File: README.md [hunk 4]
```diff
@@ -20,51 +23,211 @@ const db = require('knex')(config.database)
...
+
+        # These pass all relevant information to lolisafe.
+        proxy_set_header    Host                $host;
+        proxy_set_header    X-Real-IP           $remote_addr;
+        proxy_set_header    x-Forwarded-For     $proxy_add_x_forwarded_for;
+        proxy_set_header    x-Forwarded-Proto   $scheme;
+    }
+}
+
+# Close vi by pressing ESC + :q! + ENTER.
```
Comment (alucard0134) [inline README.md]: I would just say "open the file with your favorite editor"

if they cant fucking decide that then they shouldn't be setting up a clone

File: README.md [hunk 5]
```diff
@@ -20,51 +23,211 @@ const db = require('knex')(config.database)
...
+
+        index   home.html;              # This is the file to be sent to the requested user when they go to your domain (You shouldn't need to change this).
+
+        client_max_body_size    512M;   # This must be grater than, or equal to the max upload size you set in the lolisafe configuration.
+
+        location / {
+            proxy_pass  https://subdomain.domain.com:9999;  # This is the domain and port number you set in the lolisafe config.js file.
+
+            # These pass all relevant information to lolisafe.
+            proxy_set_header    Host                $host;
```
Comment (alucard0134) [inline README.md]:
```
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header Host $http_host;
        proxy_set_header X-NginX-Proxy true;
        proxy_pass http://127.0.0.1:9999/;
        proxy_redirect off;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
        proxy_redirect off;
```

File: README.md [hunk 6]
```diff
@@ -20,51 +23,211 @@ const db = require('knex')(config.database)
...
+    ```
+
+1. Make sure your configuration is correct with: `sudo nginx -t`.  If all goes well, you can restart NGINX with: `sudo systemctl restart nginx`. If not, make sure you've spelt everything *exactly* and aren't missing any semicolons.
+
+1. Now you're all set to [start lolisafe](#running)!
 
 ## Author
 
-**loli-safe** © [Pitu](https://github.com/Pitu), Released under the [MIT](https://github.com/WeebDev/loli-safe/blob/master/LICENSE) License.<br>
+**loli-safe** © [Pitu](https://github.com/Pitu), Released under the [MIT](https://github.com/WeebDev/loli-safe/blob/master/LICENSE) License.
```
Comment (alucard0134) [inline README.md]: at the very bottom

Comment (RyoshiKayo) [inline README.md]: That's there so markdownlint don't have a fuss about having HTML

**Comment by RyoshiKayo** [comment]
fuck it. I'm not even going to try

File: README.md (hunk 6 above)
Comment (alucard0134) [inline README.md]: markdownlint can suck my left nut

File: README.md (hunk 3 above)
Comment (Aareksio) [inline README.md]: Could you explain?

**Pull request closed without merging**